proprietary/react-icons/README.md

```diff
@@ -19,13 +19,11 @@ npm install @amsterdam/design-system-react-icons
 
 Import the component for the icon you need, as well as the generic Icon component, and use them in your JSX.
 
```

Comment:
Maybe we can ignore Prettier here, to keep the old formatting? Specifically the single quotes and not using semicolons.

Reply:
Done